x-pack/plugins/security_solution/server/lib/detection_engine/routes/signals/open_close_signals_route.ts

```diff
@@ -52,9 +53,14 @@ export const setSignalsStatusRoute = (router: IRouter) => {
           },
         };
       }
+      let onConflict = 'abort';
+      if (conflicts != null) {
+        onConflict = conflicts;
+      }
```

Comment:
What do you think about removing the above lines and replacing line 63 below with the following?

```node.js
conflicts: conflicts ?? 'abort'
```

This would allow us to remove the `let`.

Reply:
Done, thanks!